### ISSUE TITLE
ERROR

### ISSUE THREAD

**Issue by hope-creator** [lay295/TwitchDownloader#1221]
### Checklist

- [X] I have checked the [issue page](https://github.com/lay295/TwitchDownloader/issues?q=is%3Aissue) for duplicates
- [X] I am running the latest version ([download here](https://github.com/lay295/TwitchDownloader/releases/latest))

### Edition

Unsure

### Describe your issue here

ERROR: The request was canceled due to the configured HttpClient.Timeout of 30 seconds elapsing. 

I can't download anything because of this error.

**Comment by hope-creator** [comment]
![Screenshot_1](https://github.com/user-attachments/assets/859eeaff-dab1-4ee4-9f42-6a251b3bec69)
![Screenshot_2](https://github.com/user-attachments/assets/58cbedf0-cdd5-418a-b7ac-556d19d2bdba)

**Comment by ScrubN** [comment]
The issue was with Twitch servers. Try again now and see if it works again.

**Comment by hope-creator** [comment]
no, everything still doesn’t work, the program hasn’t been working for about a week now, I don’t think that the Twitch servers haven’t been working all this time.

**Comment by ScrubN** [comment]
I am able to download your VOD just fine. My theories are:
- You are using a proxy/vpn that is disallowed by the kotlin Twitch mobile app API (the website API works better with proxies/vpns but is much harder for us to fool)
- Twitch servers are having issues in your area
- Twitch servers are refusing to serve your area/Twitch banned your IP
